INSTALL.md

```diff
@@ -93,8 +93,31 @@ To uninstall Picard again simply remove all the files listed in
 Running From the Source Tree
 ----------------------------
 
-If you want to run Picard from the source directory without installing, you
-need to build the C extensions and locales manually:
+If you want to run Picard from the source directory without installing, or want to develop,
+you need to follow those steps.
+
+On Debian-based systems:
+
+   apt install python3-pyqt5 python3-venv python3-dev
+
```

Comment:
Makes sense. The Qt situation is a bit special and this used to work. Only reason is that libssl1.1 got dropped completely, and the Qt5 binaries provided by PyQt5 via pip are still built against it.

That libssl1.1 is dropped from some distros but still needed for compatibility with older ones makes distributing the Qt binaries currently difficult for sure, and there is no good choice to be made by the PyQt team. But eventually this problem will go away again.

I am currently not sure how this behaves on other distros. I think Fedora deprecated libssl1.1 as well, but it is still available for installation.

For now I think it makes sense to keep the docs as you did it. Should there be a note along the lines "For other distributions check your distribution's documentation on how to install the required packages"?